modules/core/src/v2/pendingApproval.ts

```diff
@@ -61,18 +62,21 @@ export interface ApproveOptions {
   otp?: string;
   tx?: string;
   xprv?: string;
+  txRequestId?: string;
```

Comment:
move this out from `ApproveOptions` and into `PendingApprovalData` - wallet platform will return this info if the pending approval is associated to a tx request